akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala

```diff
@@ -59,7 +62,7 @@ private[cluster] object StressMultiJvmSpec extends MultiNodeConfig {
 
   val totalNumberOfNodes =
     System.getProperty("MultiJvm.akka.cluster.Stress.nrOfNodes") match {
-      case null  => 13
+      case null  => 26
```

Comment:
I will revert most changes in this test before merging, but wanted to show what I used for testing this. Downing 12 of 26 nodes.

Reply:
👍